src/reducers/RecipeRecommendationReducer.js

```diff
@@ -0,0 +1,21 @@
+import { recipeRecommendationActionTypes as actionTypes } from '../constants/actionTypes';
```

Comment:
This is probably fine, but it should be part of the existing RecipeReducer

Reply:
yes I agree, it should follow the same pattern

Reply:
Does it matter?

Reply:
I would argue that it is the same